lib/constants/category.ts

```diff
@@ -0,0 +1,24 @@
+export type Category =
+  | 'ansible'
+  | 'batect'
+  | 'bazel'
+  | 'c'
+  | 'cd'
+  | 'ci'
+  | 'dart'
+  | 'docker'
+  | 'dotnet'
+  | 'elixir'
+  | 'golang'
+  | 'helm'
+  | 'iac'
+  | 'java'
+  | 'javascript'
+  | 'kubernetes'
+  | 'node'
+  | 'php'
+  | 'python'
+  | 'ruby'
+  | 'rust'
+  | 'swift'
+  | 'terraform';
```

Comment:
Are there any of the existing `language` values we have which are *not* in this list?

Reply:
No, there are all there. Just `js` has been replaced with `javascript`.

Reply:
Can we keep it in the interests of backwards compatibility for later when we deprecate `languages`? Or you strongly feel `javascript` is correct?

Reply:
No strong feelings regarding this, it simply comes across clearer for me.

Reply:
In that case I'd prefer to keep it as `js` so we have less to think about when we deprecate languages